bin/p2-setup-runtime/main.go

```diff
@@ -0,0 +1,191 @@
+package main
+
+import (
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	dockerclient "github.com/docker/docker/client"
+	"github.com/square/p2/pkg/hooks"
+	"github.com/square/p2/pkg/logging"
+	"github.com/square/p2/pkg/manifest"
+	"github.com/square/p2/pkg/pods"
+	"github.com/square/p2/pkg/types"
+	"github.com/square/p2/pkg/uri"
+	"github.com/square/p2/pkg/version"
+	"gopkg.in/alecthomas/kingpin.v2"
```

Comment:
group the imports via goimports

Reply:
file was already formatted using `goimports`. Is there any change  you are expecting ?